OJP/OJP_FareSupport.xsd

```diff
@@ -334,7 +334,14 @@
 			</xs:element>
 			<xs:element name="SaleUrl" type="WebLinkStructure" minOccurs="0" maxOccurs="unbounded">
 				<xs:annotation>
-					<xs:documentation>URL to buy the FareProduct online</xs:documentation>
+					<xs:documentation>URL to buy the FareProduct online. 
+					
+					Note: EU-Spirit: 
```

Comment:
What do you mean here: is this specific to data produced for EU-Spirit ? It looks like a part of a profile put in the main standard document.

Reply:
=> EU-Spirit behaviour will be moved to the EU-Spirit profile

Reply:
only remove line 339